### ISSUE TITLE
[SUPPORT] Search for exact term

### ISSUE THREAD

**Issue by DutchPete** [pbek/QOwnNotes#3190]
#### Question
From https://www.qownnotes.org/getting-started/searching.html I understand that `"note book"` will search for notes with exactly the text note book in the file name or note text.
This is not the case.

Real life example: I wanted to see if I had info about TWA, the historic U.S. airline. So, after hitting `Alt+F`, I typed `"twa"` (exactly like that) in the search box.

1 of the results was a note with the word *postwar*. Note there is twa in that word.
So, the statement on your web page that *will search for notes with exactly the text note book in the file name or note text* does not seem to be a true statement. Even typing "TWA" in the search box still return *postwar*.

What am I doing wrong?

**Comment by pbek** [comment]
> "note book" will search for notes with exactly the text note book in the file name or note text

"Exactly" doesn't mean it does a word-boundary search!

**Comment by DutchPete** [comment]
You have not answered my question, so let me rephrase it:

in the example I gave above, what is the search parameter I should use get a return with notes that show the standalone word *TWA* or *twa*, NOT as part of another word.

I do these kinds of searches often so it is important to me to know how to do it.
Or are you telling me it cannot be done in QON?

**Comment by pbek** [comment]
There is no word boundary search. That's correct.

**Comment by DutchPete** [comment]
Feature Request #3191 submitted.

**Comment by Kvcd12** [comment]
I am using version 24.3.5.  If you use Replace instead of Find, there is a drop down box labeled Mode.  The drop down choices are Plain text, Whole words, and Regular expression.  Using your postwar TWA example, choosing Whole words only found TWA.

**Comment by pbek** [comment]
Yeah, but that only works in the current note, not over all notes.

**Comment by DutchPete** [comment]
I'm not reopening this issue, but just want to let you know that this is a show stopper for me.

The whole point of being able to create 1000s of notes is to have the certainty of being able to find something, anything in that mountain of info. Not being able to do a global search through all those notes makes the info base literally useless.

Perhaps it is just me, or perhaps I am part of a tiny minority who needs this, in which case it is not worth spending your time on. In any case, it is unacceptable for me.
